contracts/ToucanRegenBridge.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier:  GPL-3.0
 
-pragma solidity ^0.8.4;
+pragma solidity ^0.8.14;
```

Comment:
@bezerrablockchain please remove the caret from here and all other files so these contracts are pinned to 0.8.14.
```suggestion
pragma solidity 0.8.14;
```

Reply:
Done my friend